scripts/podman-volume-backup.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# Get a list of all volumes
+volumes=$(podman volume ls -q)
+#get timestamp for unique filename
+timestamp=$(date +"%Y%m%d%H%M%S")
+
+# Pause all running containers
+echo "Pausing containers for backup"
+podman pause $(podman ps -q)
```

Comment:
Please change this to 
`podman pause --all`

Reply:
done